types/types.go

```diff
@@ -49,6 +58,6 @@ type SignalSourceComplete string
 //Finding is the main output of a signature. It represents a match result for the signature business logic
 type Finding struct {
 	Data        map[string]interface{}
-	Context     Event
+	Context     interface{}
```

Comment:
Maybe it's a good time to rename Context to Event or Payload? IMO Context is quite confusing.

Reply:
Maybe, throughout the project we usually assign the event payload here so I guess `Payload` would be more appropriate.
I was also considering a more radical approach: we could report Findings through the new `Event` type, for example by setting the content type to `tracee/finding`.
This way we could have tracee responds to it's own findings organically (in terms of use cases - perhaps some signatures could depend on another signature triggering it, etc.) WDYT?

Reply:
That's also a good idea. If we're not sure I'd leave it as is for now. We can maybe just document that TraceeEvents represents a payload of the event generated by tracee-ebpf to distinguish it from Event envelope, i.e. headers + payload

Reply:
I think i'll leave as is, maybe after this is merged we can open an issue about using these new capabilities more thoroughly.
For now - I'll document what you requested.

Reply:
Resolved in #1493